frontend/src/components/layout/SignUp.jsx

```diff
@@ -0,0 +1,31 @@
+
+import styles from './SignUp.module.css'
+
+const SignUp = () =>{
+    return (
+        <>
```

Comment:
We don't need a fragment here since it only has one child element (the div).

Reply:
Also, the indentation/formatting here is not quite right. Please make sure to install the `Prettier` extension and turn on `Autosave` and `Format on Save` in your settings.